website/content/docs/upgrading/instructions/general-process.mdx

```diff
@@ -46,125 +44,128 @@ If you are using Kubernetes, then please review our documentation for
 
 ## Prepare for the Upgrade
 
-**1.** Take a snapshot:
+1. Take a snapshot to ensure you have a safe fallback option in case something goes wrong.
 
-```
-consul snapshot save backup.snap
-```
+  ```shell-session
+  $ consul snapshot save backup.snap
+  ```
 
-You can inspect the snapshot to ensure if was successful with:
+  You can inspect the snapshot to ensure your cluster's Raft index was successfully captured.
 
-```
-consul snapshot inspect backup.snap
-```
+  ```shell-session
+  $ consul snapshot inspect backup.snap
+  ```
 
-Example output:
+  Example output:
 
-```
-ID           2-1182-1542056499724
-Size         4115
-Index        1182
-Term         2
-Version      1
-```
+  ```shell-session hideClipboard
+  ID           2-1182-1542056499724
+  Size         4115
+  Index        1182
+  Term         2
+  Version      1
+  ```
 
-This will ensure you have a safe fallback option in case something goes wrong. Store
-this snapshot somewhere safe. More documentation on snapshot usage is available here:
+  Store this snapshot somewhere safe. For more information on snapshots, refer to the following:
 
-- [consul.io/commands/snapshot](/consul/commands/snapshot)
-- [Backup Consul Data and State tutorial](/consul/tutorials/production-deploy/backup-and-restore)
+  - [`consul snapshot` CLI command](/consul/commands/snapshot)
+  - [Backup Consul Data and State tutorial](/consul/tutorials/production-deploy/backup-and-restore)
 
-**2.** Temporarily modify your Consul configuration so that its [log_level](/consul/docs/agent/config/cli-flags#_log_level)
-is set to `debug`. After doing this, issue the following command on your servers to
+2. Temporarily modify your Consul configuration to set the agent's [`log_level`](/consul/docs/agent/config/cli-flags#_log_level) is set to `debug`. Then issue the following command on your servers to
 reload the configuration:
 
+```shell-session
+$ consul reload
 ```
-consul reload
+
+When you change the cluster's log level, Consul gives you more information to work with in the event that something goes wrong.
+
+### Enterprise upgrades
+
+If your datacenter runs Consul Enterprise, update your server agent configuration file to disable autopilot's upgrade migration:
+
+```shell-session
+$ consul operator autopilot set-config -disable-upgrade-migration=true
 ```
 
-This change will give you more information to work with in the event something goes wrong.
+When you disable the upgrade migration feature on Consul Enterprise, you experience a smoother upgrade process. Consul Enterprise supports [automated upgrades](/consul/docs/enterprise/upgrades), but the autopilot feature may cause a node running an updated Consul version to elect a new leader before the version is updated on the existing cluster leader.
```

Comment:
It would be more effective to highlight this as a warning,  similar to the ent tutorial upgrade.